apps/docs-app/docs/experiments/sfc/index.md

```diff
@@ -0,0 +1,344 @@
+---
+sidebar_position: 1
+---
+
+# Analog SFCs
+
+> **Note:**
+>
+> This file format and API is extremely experimental and is a community project, not an official Angular proposal API. Use it at your own risk.
+
+The `.analog` file extension denotes a new file format for Angular Single File Components (SFCs) that aims to simplify the authoring experience of Angular components.
+
+Together, it combines:
+
+- Colocated template, script, and style
+- No decorators
+- Performance-first defaults (`OnPush` change detection, no accesss to `ngDoCheck`, etc.)
+
+Here's a demonstration of the Analog format building a simple todo list:
+
+```html
+<script lang="ts">
+  import { signal } from '@angular/core';
+
+  let id = 0;
+
+  interface TodoItem {
+    number;
+    string;
+    boolean;
```

Comment:
Question: Is this a new way to write interfaces in newer TypeScript versions? I haven't seen this before, and my Googling skills aren't helping 😅
```suggestion
    id: number;
    name: string;
    done: boolean;
```

Reply:
It's not :( Webstorm's autoformat seemingly broke this MD file's code samples worse than I thought. Sorry about this

Reply:
No worries! I genuinely thought this was some new syntax as I haven't been up to date recently.